### PR TITLE
Feat: standard slice

### PR DESCRIPTION
**Pull request by ValGeorgiev**
Implement standard slice for mobile and tablet 

Mobile:
image

Tablet:
image

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-k/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+} from "../shared";
+import styles from "./styles";
+
+const TileK = ({ onPress, tile }) => (
+  <TileLink onPress={onPress} tile={tile}>
+    <View style={styles.container}>
+      <TileImage
+        aspectRatio={3 / 2}
+        style={styles.imageContainer}
+        uri={getCrop(tile.leadAsset || tile.article.leadAsset, "crop32")}
```
Comment (nehasri89) [inline packages/edition-slices/src/tiles/tile-k/index.js]: should do tile.leadAsset || tile.article.listingAsset || tile.article.leadAsset
as we found out lately and in a PR https://github.com/newsuk/times-components/pull/1826

File: packages/edition-slices/src/slices/magazine/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        renderSupport3={() => (
+          <TileK onPress={onPress} tile={support3} tileName="support3" />
+        )}
+        renderSupport4={() => (
+          <TileK onPress={onPress} tile={support4} tileName="support4" />
+        )}
+      />
+    )}
+    renderSmall={breakpoint => (
+      <MagazineSlice
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/magazine/index.js]: this can be DRYed up

File: packages/edition-slices/src/tiles/tile-k/index.js (hunk 1 above)
Comment (ValGeorgiev) [reply packages/edition-slices/src/tiles/tile-k/index.js]: Good catch, thanks. I think we need to add this in a shared file and reuse it everywhere. I will create a PR when both PRs are merged

File: packages/edition-slices/src/slices/magazine/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <ResponsiveSlice
+      renderMedium={breakpoint => (
+        <MagazineSlice
+          breakpoint={breakpoint}
+          onPress={onPress}
+          renderTiles={tiles}
+        />
+      )}
+      renderSmall={breakpoint => (
+        <MagazineSlice
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/magazine/index.js]: can DRY this up too.

Comment (aronshamash) [reply packages/edition-slices/src/slices/magazine/index.js]:
```  
const renderSlice = breakpoint => (
        <MagazineSlice
          breakpoint={breakpoint}
          onPress={onPress}
          renderTiles={tiles}
        />
  );
``` 
?

File: packages/fixture-generator/src/types.ts [hunk 4]
```diff
@@ -1460,6 +1460,11 @@ export interface LeadOneAndFourSlice extends ArticleSlice {
 
   sections: Section[];
 }
+export interface MagazineSlice extends ArticleSlice {
```
Comment (tuncaulubilge) [inline packages/fixture-generator/src/types.ts]: We should just use `StandardSlice`, we don't need a special MagazineSlice

File: packages/edition-slices/src/edition-slices.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ import {
   DailyRegisterLeadFourSlice,
   LeadersSlice,
   LeadOneAndFourSlice,
+  MagazineSlice,
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/edition-slices.js]: StandardSlice please

**Comment by tuncaulubilge** [comment]
We shouldn't need "magazine" name in anywhere in this pr. The whole thing should be StandardSlice and some new Tile

File: packages/edition-slices/src/slices/index.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 export { default as CommentLeadAndCartoonSlice } from "./commentleadandcartoon";
 export { default as DailyRegisterLeadFourSlice } from "./dailyregisterleadfour";
 export { default as LeadOneAndFourSlice } from "./leadoneandfour";
+export { default as MagazineSlice } from "./magazine";
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/slices/index.js]: export { default as StandardSlice } from "./standard";

**Comment by aronshamash** [comment]
@ValGeorgiev should probably update the PR title too

File: packages/edition-slices/edition-slices.showcase.js [hunk 7]
```diff
@@ -70,9 +72,14 @@ const sliceStories = [
   },
   {
     mock: mockLeadOneAndFourSlice(),
-    name: "Lead One And Four (I,J,J,J,J)",
+    name: "Lead One And Four (Mobile: I,J,J,J,J, Tablet: AC,AD,AD,AD,AD)",
     Slice: LeadOneAndFourSlice
   },
+  {
+    mock: mockStandardSlice(),
+    name: "Standard Slice (K)",
```
Comment (tuncaulubilge) [inline packages/edition-slices/edition-slices.showcase.js]: Just a suggestion, we can make it `K*` to indicate variable number of tiles
"    name: "Standard Slice (K*)",

File: packages/edition-slices/src/slices/standard/index.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { View } from "react-native";
+import PropTypes from "prop-types";
+import { Column } from "@times-components/slice-layout";
+import { TileK } from "../../tiles";
+import { ResponsiveSlice } from "../shared";
+import styles from "./styles";
+
+const Standard = ({ onPress, slice: { items } }) => {
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/slices/standard/index.js]: This needs an update following the slice refactor. Can you convert this into a Component?

File: packages/edition-slices/edition-slices.showcase.js (hunk 7 above)
Comment (ValGeorgiev) [reply packages/edition-slices/edition-slices.showcase.js]: done

File: packages/edition-slices/src/slices/standard/index.js (hunk 8 above)
Comment (ValGeorgiev) [reply packages/edition-slices/src/slices/standard/index.js]: done

File: packages/edition-slices/src/slices/standard/index.js [hunk 9]
```diff
@@ -1,29 +1,37 @@
...
-const Standard = ({ onPress, slice: { items } }) => {
-  const tiles = items.map(tile => () => (
-    <TileK onPress={onPress} tile={tile} />
-  ));
+class Standard extends Component {
+  constructor(props) {
+    super(props);
 
-  const renderMediumSlice = () => (
+    this.tiles = props.slice.items.map(tile => () => (
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/slices/standard/index.js]: ??

**Comment by ValGeorgiev** [comment]
All review comments should be fixed now, let me know if you have any issues with the implementation and I we can discuss and change it in my next PR about magazine

File: packages/edition-slices/src/slices/standard/index.js [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+  getTiles() {
+    const {
+      slice: { items },
+      onPress
+    } = this.props;
+
+    return items.map(tile => () => <TileK onPress={onPress} tile={tile} />);
+  }
+
+  renderMedium = () => (
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/slices/standard/index.js]: Bind this on the constructor. This is a big repo and last thing we want is inconsistency

**Pull request merged**